src/plugins/intel_cpu/src/node.h

```diff
@@ -111,6 +112,34 @@ class NodeDesc {
         executorFactory = factory;
     }
 
+    bool hasZeroInputDims() const {
+        const auto& inputConfigs = getConfig().inConfs;
+
+        return std::any_of(inputConfigs.begin(), inputConfigs.end(), [](const PortConfig& portConfig) {
+            return portConfig.hasZeroDims();
+        });
```

Comment:
This is my first version.
Trying to avoid an access by index where possible

Reply:
The idea of my suggestion is to reuse the existing method, increasing code reuse, as long as `hasZeroInputDimsAtPort` exists anyway. And what is the problem with the index usage, inputConfigs is an `std::vector` so access by index is not expensive.

Reply:
This is not about performance rather using modern style where possible to avoid index addressing related issues.
This is completely minor discussion; I propose to not waste time on this. We have much bigger problems to fix.